src/ui/page/accessibility-statement.html

```diff
@@ -40,30 +40,36 @@ <h2 class="heading-medium">Reporting accessibility problems with this website</h
-                    AA standard, due to the non-compliances and exemptions listed below.</p>
+                <p>Government Business Services is committed to making this website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+                
+                <h3 class="heading bold-small">Compliance status</h3>
+                <p>This website is partially compliant with the <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1</a> AA standard, due to the non-compliances and exemptions listed below.</p>
 
                 <h3 class="heading bold-small">Non-accessible content</h3>
                 <p>The content listed below is non-accessible for the following reasons.</p>
 
                 <h4 class="heading bold-small">Non-compliance with the accessibility regulations</h4>
-                <p>The course search filters are not operable for users navigating with only a keyboard. This fails WCAG 2.1 success criterion 2.1.1 (keyboard).</p>
-                <p>We plan to implement a solution for this by October 2020.</p>
+                <ol class="list-number">
+                    <li>Some forms controls are not detectable by screen reader software - this fails WCAG 2.1 success criterion 4.1.2 (Name, Role, Value)</li>
+                </ol>
 
                 <h2 class="heading-medium">Content that’s not within the scope of the accessibility regulations</h2>
                 <h3 class="heading bold-small">Online learning packages</h3>
-                <p>The online learning packages hosted on the Civil Service Learning website are created by external suppliers. Government Automation and Business Services and Civil Service HR in
+                <p>The online learning packages hosted on the Civil Service Learning website are created by external suppliers. Government Business Services and Civil Service HR in
                     the Cabinet Office work closely with these suppliers to ensure that all learning is accessible
                     for users. The Cabinet Office is not legally responsible for these learning packages meeting
                     the WCAG 2.1 AA criteria.</p>
 
+                <h2 class="heading-medium">What we're doing to improve accessibility</h2>
+                <p>We are urgently fixing content which fails to meet the Web Content Accessibility Guidelines version 2.1 AA standard. We will update this page when issues are fixed.</p>
+
                 <h2 class="heading-medium">Preparation of this accessibility statement</h2>
-                <p>This statement was prepared on 23 September 2020. It was last reviewed on 23 September 2020.</p>
-                <p>Last updated: 23 September 2020</p>
+                <p>This statement was prepared on 15 February 2021. It was last reviewed on 15 February 2021.</p>
```

Comment:
The last update date should be the date of the go-live.

Reply:
Do you mean last reviewed here?